components/kyma-environment-broker/internal/process/update/btp_operator_overrides.go

```diff
@@ -31,14 +31,14 @@ func (s *BTPOperatorOverridesStep) Name() string {
 	return "BTPOperatorOverrides"
 }
 
-func (s *BTPOperatorOverridesStep) Run(operation internal.UpdatingOperation, logger logrus.FieldLogger) (internal.UpdatingOperation, time.Duration, error) {
+func (s *BTPOperatorOverridesStep) Run(operation internal.Operation, logger logrus.FieldLogger) (internal.Operation, time.Duration, error) {
 	// get btp-operator component input and calculate overrides
 	ci, err := getComponentInput(s.components, BTPOperatorComponentName, operation.RuntimeVersion, operation.InputCreator.Configuration())
 	if err != nil {
-		return s.operationManager.OperationFailed(operation, "failed to get components", err, logger)
+		return s.operationManager.RetryOperation(operation, "failed to get components", err, 5*time.Second, 30*time.Second, logger)
```

Comment:
nit: I think given the component input is from a mounted `ConfigMap`, requeue is unlikely to help but 30 sec timeout doesn't hurt much either